repos/system_upgrade/el7toel8/actors/restrictedpcisscanner/libraries/restrictedpcisscanner.py

```diff
@@ -0,0 +1,105 @@
+import json
+import os
+
+import urllib3
+
+from leapp.libraries.common.config import get_env
+from leapp.libraries.stdlib import api
+from leapp.models import RestrictedPCIDevices
+
+REQUEST_TIMEOUT = 0.5
+CUR_DIR = os.path.dirname(os.path.abspath(__file__))
+UNSUPPORTED_DRIVER_NAMES_FILE = os.path.join(
+    CUR_DIR, "..", "data", "unsupported_driver_names.json"
+)
+UNSUPPORTED_PCI_IDS_FILE = os.path.join(
+    CUR_DIR, "..", "data", "unsupported_pci_ids.json"
+)
+
+
+def get_restricted_pcis(http, update_local_data=False):
+    """Get data about unsupported driver names and unsupported pci_ids.
+
+    The function makes API call to the web service, which provides
+    these data.
+
+    :param urllib3.PoolManager http: pool manager instance
+    :param bool update_local_data: if True writes the received data to
+        the corresponding json data files (inside ../data dir)
+
+    :returns: Tuple[dict, dict]
+    """
+    # TODO this constant should be above the scope of this func.
+    #   However due to the way how actor tested with CurrentActorMocked
+    #   we can't do this.
+    API_URL = (
+        "http://"
+        + get_env(
+            "LEAPP_DEVEL_UNSUPPORTED_PCIS_HOST", default="10.0.79.153:8000"
```

Comment:
should I rename the env var LEAPP_DEVEL_UNSUPPORTED_PCIS_HOST -> LEAPP_DEVEL_UNSUPPORTED_PCIS_HOST_PORT, or add additional env var LEAPP_DEVEL_UNSUPPORTED_PCIS_PORT, or leave as it is now

Reply:
I would say either just "DEVEL" if it is our debug/testing variable or UNSUPPORTED if it is supposed to be used also by users but is unsupported.

Concerning the hostname and port, I am inclined to use just HOST and document it properly.

Reply:
Considering https://leapp.readthedocs.io/en/latest/best-practices.html#use-the-leapp-and-leapp-devel-prefixes-for-new-envars the right choice here is:
LEAPP_DEVEL_UNSUPPORTED_PCIS_HOST
Agree?

Reply:
Ok, so my previous comment was probably more like I imagine it in my head. 

Then the correct name would be `LEAPP_DEVEL_PCIS_HOST`. `LEAPP_UNSUPPORTED` is just name of the single envvar to enable `DEVEL` vars.

Reply:
OK, I will fix that, shorter -> better now:)

Reply:
Solved in this MR (https://github.com/oamg/leapp-repository/commit/e73518a83d1dced4557b967b773971e025b62fe1#diff-1f6ae31d29d635b664336b8bac1383ecR41)